### DISCUSSION TITLE
Migration to wiki - import comments ?

### DISCUSSION THREAD

**Discussion by nathalie-beaudoin** [requarks/wiki#5708] [Help / Questions] [answered]
Hi !

I am presently migrating from Confluence to wiki.  The comments are not natively exported in Confluence, but I think that a tool like this one can make it: https://marketplace.atlassian.com/apps/284/confluence-command-line-interface-cli?tab=overview&hosting=cloud

With that said, I am able to export the comments from Confluence, is there a way to import it in wiki.js during my migration ?

Thanks, Nathalie

**Comment by NGPixel** [comment, marked as answer]
You would need to import them directly into the DB via SQL. There's no import tool from Wiki.js itself.
